DOC-468 | BTS-1341 | ArangoSearch Views building in progress query warning (#1361)
arangodb/docs: 3.11/release-notes-api-changes311.md

shards. This value can be higher than the configured `numberOfShards` value of
the graph due to internally used hidden collections.

#### Cursor API

When you link a collection to an `arangosearch` View and run an AQL query

> against this View while it is still being indexed, you now receive a query
> warning. This warning alerts you about potentially incomplete results obtained

Reviewer: ```suggestion
against this View while it is still being indexed, you now receive the query result
including a warning. This warning alerts you about potentially incomplete results obtained
```

You can wrap calls to these methods with a `try { ... }` block to catch errors,
for example, in _arangosh_ or in Foxx services.

> ### Query methods

Reviewer: ```suggestion
### Query method
```

> When using the `db._query` method to execute an AQL query against an

Reviewer: ```suggestion
When you use the `db._query()` method to execute an AQL query against an
```

`arangosearch` View while it is still in the process of being built,

> the query now includes a warning message that the result set may not be

Reviewer: ```suggestion
the query now includes a warning message that the results may not be
```

> complete due to the ongoing building process of the View.

Reviewer: ```suggestion
complete due to the ongoing indexing process of the View.
```
